internal/compiler/wasm/optimizations.go

```diff
@@ -176,7 +177,11 @@ func (c *Compiler) removeUnusedCode() error {
 	// anything referenced in a table
 	for _, seg := range c.module.Element.Segments {
 		for _, idx := range seg.Indices {
-			reach(cgIdx, keepFuncs, idx)
+			if c.skipElem(keepFuncs, idx) {
+				c.debug.Printf("skipping elem %d", idx)
```

Comment:
nit: I wonder if a more explicit message would be nice? `dropping element %d because policy does not depend on re2`